WcaOnRails/app/assets/stylesheets/registrations.scss

```diff
@@ -25,3 +26,22 @@ table.registrations-table {
   z-index: 1000; /* show over sticky table headers */
   bottom: calc(5px + #{$footer-height-px});
 }
+
+/* Styling bootstrap table sort */
+.fixed-table-loading {
+  display: none;
+}
+
+th {
+  &:focus { outline: none; }
+  .sortable {
+    &:hover { cursor: pointer; }
```

Comment:
Just as a hard rule, please always put css rules on their own line. Who knews when we'll need to add something to this.